api/src/main/java/me/lokka30/treasury/api/economy/misc/APIUtils.java

```diff
@@ -0,0 +1,19 @@
+package me.lokka30.treasury.api.economy.misc;
+
+// TODO: What shall we call this?
+public class APIUtils {
+
+    /**
+     * @author lokka30
+     * @since v1.0.0
+     * If the specified amount is less than zero
+     * then zero is returned. Otherwise, the amount
+     * is returned. This ensures an amount is at least zero
+     * since negative values are not allowed in the API.
+     * @param amount to check for.
+     * @return the unmodified or modified amount.
+     */
```

Comment:
This javadoc is icorrect. The correct order is first description then tags. If not the javadoc parser thinks that the description is, in this case, inside the since tag

Reply:
1. This is going to be long until it is completed
2. This has been copied like that.

Reply:
Yeah I've been checking other files and most of them have this problem. I've also though that maybe the \@since tag should have a link to the EconomyAPIVersion corresponding to it, I'll mention this on #27 to see what others think

Reply:
there will be a separate PR to fix all javadoc problems.